css/styles.css

```diff
@@ -1878,6 +1884,136 @@ img {
 
 }
 
+.search-bar {
+    width: 330px;
+}
+
+.search-bar .search-form {
+    margin-bottom: 21px;
+}
+
+.search-bar .catalog-input {
+    width: 330px;
+}
+
+.categories,
+.last-posts,
+.tags {
+    background-color: #141B22;
+}
+
+.categories,
+.last-posts {
+    margin-bottom: 40px;
+}
+
+.search-bar-subtitle {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+    text-align: left;
+    text-transform: uppercase;
+    color: #FFFFFF;
+    padding: 16px 24px;
+    border-bottom: 1px solid #434343;
+}
+
+.categories-list {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    padding: 12px 24px;
+}
+
+.categories-link {
+    font-size: 14px;
+    line-height: 22px;
```

Comment:
I would advise setting the `line-height` in dimensionless quantities - here it's 1.57 (22px / 14px). Would be good if you will do it whole project